codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  branch: dev
+
+coverage:
```

Comment:
Is it covering only the unit tests, only integration tests, or both?

Reply:
For now only the unit tests.
There are some more complex solutions for merging the coverages from both, but I wanted to make sure the tests with the most coverage would be already available now.